testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -558,6 +559,19 @@ def test_Cluster_add_metadata(self, cluster):
         assert np.all(cluster.metadata['test'] == metadata), \
                      "Cluster metadata isn't as expected"
 
+    def test_empty_Cluster(self):
```

Comment:
```suggestion
    def test_empty_cluster(self):
```
I see the intent behind the capitalization here, however it's best to keep to pep8 here if possible: https://peps.python.org/pep-0008/#function-and-variable-names